lib/features/thread/presentation/styles/scroll_to_top_button_widget_styles.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:core/core.dart';
+import 'package:flutter/material.dart';
+
+class ScrollToTopButtonWidgetStyles {
+  static const double defaultLimitIndicator = 50.0;
+  static const double defaultElevation = 5.0;
+  static const Color defaultColor = AppColor.colorLabelCancelButton;
+  static const double defaultButtonRadius = 44.0;
+  static const double buttonPadding = 16.0;
```

Comment:
IMO it is 8 

<img width="190" alt="Screenshot 2023-10-10 at 05 07 12" src="https://github.com/linagora/tmail-flutter/assets/6462404/f12a0886-323a-4683-ac9f-7bafcdd5e3b1">

Reply:
I changed dimension of button. Please see videos above!